### PR TITLE
Do not include url scheme in refs.Org

### PR DESCRIPTION
**Pull request by fejta**
Also allow presubmits to leave scheme off the repo key.

/assign @krzyzacy @BenTheElder

### CODE REVIEW

File: prow/gerrit/adapter/adapter.go [hunk 1]
```diff
@@ -168,20 +168,20 @@ func (c *Controller) Sync() error {
 	return nil
 }
 
-func makeCloneURI(instance, project string) (string, error) {
+func makeCloneURI(instance, project string) (*url.URL, error) {
 	// TODO(fejta): drop the https:// part
```
Comment (krzyzacy) [inline prow/gerrit/adapter/adapter.go]: you can resolve this todo now

Comment (fejta) [reply prow/gerrit/adapter/adapter.go]: Done

**Pull request merged**